src/main/java/org/isf/patient/model/Patient.java

```diff
@@ -174,8 +174,8 @@ public class Patient extends Auditable<String> {
 	private String profession;
 
 	@NotNull
-	@Column(name="PAT_DELETED")
-	private String deleted = "N";
+	@Column(name="PAT_DELETED", columnDefinition = "char(1) default 'N'")
+	private Character deleted = 'N';
```

Comment:
```suggestion
	private char deleted = 'N';
```

Same change as noted above.